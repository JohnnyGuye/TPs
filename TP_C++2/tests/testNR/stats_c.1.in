ADD 0 2015 5 8 10 5 5 J
ADD 4999 2015 5 8 10 5 5 V
ADD 0 2015 5 8 10 6 5 R
ADD 4999 2015 5 8 10 6 5 V
ADD 0 2015 5 8 10 7 5 J
ADD 4999 2015 5 8 10 7 5 J
ADD 0 2015 5 8 10 8 5 N
ADD 4999 2015 5 8 10 8 5 V
ADD 0 2015 5 8 10 9 5 J
ADD 4999 2015 5 8 10 9 5 V
ADD 0 2015 5 8 10 10 5 V
ADD 4999 2015 5 8 10 10 5 V
ADD 0 2015 5 8 10 11 5 V
ADD 4999 2015 5 8 10 11 5 V
ADD 0 2015 5 8 10 12 5 R
ADD 4999 2015 5 8 10 12 5 J
ADD 0 2015 5 8 10 13 5 N
ADD 4999 2015 5 8 10 13 5 N
ADD 0 2015 5 8 10 14 5 N
ADD 4999 2015 5 8 10 14 5 R
STATS_C 4999
EXIT
