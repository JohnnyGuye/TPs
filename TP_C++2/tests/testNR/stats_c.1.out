V 60%
J 20%
R 10%
N 10%
